### PR TITLE
feat: discord card

### PR DESCRIPTION
**Pull request by plskz**
some awesome fucking card

### CODE REVIEW

File: src/components/DiscordCard.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <Container
+      width='350px'
+      height='300px'
+      display='flex'
+      flexDirection='column'
+      backgroundColor='#18191C'
+      borderRadius={10}
+    >
+      <Avatar
+        src='https://dcdn.dstn.to/avatars/90431685472038912'
```
[the remark this answers is not in the document]
Comment (plskz) [reply src/components/DiscordCard.jsx:20]: @yrnmsk drop youka id >.>

[the remark this answers is not in the document]
Comment (yrnmsk) [reply src/components/DiscordCard.jsx:20]: > @yrnmsk drop youka id >.>

239381559482777600

**Comment by yrnmsk** [comment]
stale

**Review by yrnmsk — APPROVED** [review]

**Comment by yrnmsk** [comment]
stale

**Comment by yrnmsk** [comment]
comment

**Comment by plskz** [comment]
test

File: src/components/DiscordCard.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (plskz) [reply src/components/DiscordCard.jsx]: @yrnmsk comment

**Comment by plskz** [comment]
r

**Pull request merged**